.github/workflows/push_pr.yml

```diff
@@ -8,87 +8,34 @@ on:
   pull_request:
 
 env:
-  TAG: "v0.0.0" # needed for goreleaser windows builds
-  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
-  ORIGINAL_REPO_NAME: "newrelic/nri-jmx"
-  NRJMX_VERSION: '2.6.0'
-  DOCKER_LOGIN_AVAILABLE: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+  NRJMX_VERSION: '2.6.0' # TODO check if we should update it to latest release version
 
 jobs:
-  static-analysis:
-    name: Run all static analysis checks
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: newrelic/newrelic-infra-checkers@v1
-      - name: Semgrep
-        uses: returntocorp/semgrep-action@v1
-        with:
-          auditOn: push
-
-      - name: Install Go
-        uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
+  # reusable_push_pr contains static-analysis but it does not contain the Semgrep step
+  # static-analysis job in reusable_push_pr cannot be disabled
+  # uncommenting the below leads to the following error "creating validator: parsing markdown: parsing markdown headers: unexpected additional L1 header \"2.4.7 (2021-06-10)\" found, only a single L1 header is allowed"
+  # TODO check if we can remove the below code and ignore the Semgrep step
+  # static-analysis:
+  #   name: Run all static analysis checks
+  #   runs-on: ubuntu-20.04
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: newrelic/newrelic-infra-checkers@v1
+  #     - name: Semgrep
+  #       uses: returntocorp/semgrep-action@v1
+  #       with:
+  #         auditOn: push
```

Comment:
It was removed from other repositories. Ex [nri-postgresql](https://github.com/newrelic/nri-postgresql/pull/140/files). I'd double-check with the team if it is required here for some reason.

Reply:
If it was removed in nri-postgresql then it makes sense to remove the static-analysis step as the remaining part is present in reusable workflow.

Reply:
Hey @paologallinaharbur can you confirm if we can ignore the Semgrep step in the static-analysis job, similar to the above mentioned nri-postgresql

Reply:
removed the commented static-analysis job